west.yml

```diff
@@ -193,7 +193,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: d5e5358e56542b94bc65b6483396f50ed8f3172d
+      revision: 73620197038d7ba80fb1f9abd001828b9dd6a27e
```

Comment:
Change it to 78d1912dbc2b5f6e114d7fd19a31053716a3b01d?

Reply:
@JiafeiPan , this hash seems to be the right one for this PR.